### PR TITLE
macOS support is added

### PR DESCRIPTION
**Pull request by alex-melnyk**
Implements #205

### CODE REVIEW

File: macos/app_settings.podspec [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint app_settings.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'app_settings'
+  s.version          = '0.0.1'
```
Comment (navaronbracke) [inline macos/app_settings.podspec:7]: There are a bunch of default values here.

File: macos/Classes/AppSettingsEnums.swift [hunk 2]
```diff
@@ -0,0 +1,117 @@
+
+enum SettingsGeneral: String {
```
Comment (navaronbracke) [inline macos/Classes/AppSettingsEnums.swift:2]: Of all these enums, only two constants can be used at the moment.

You should either provide an API to open a URL with a specific value, or remove the unused enum constants.

**Review by navaronbracke — CHANGES_REQUESTED** [review]
Some minor feedback about the changes.

File: macos/app_settings.podspec (hunk 1 above)
Comment (alex-melnyk) [reply macos/app_settings.podspec]: fixed

File: macos/Classes/AppSettingsEnums.swift (hunk 2 above)
Comment (alex-melnyk) [reply macos/Classes/AppSettingsEnums.swift:1]: fixed

File: macos/app_settings.podspec [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# Run `pod lib lint app_settings.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'app_settings'
+  s.version          = '5.1.1'
+  s.summary          = 'A Flutter plugin for opening macOS desktop settings from an app.'
+  s.description      = 'A Flutter plugin for opening macOS desktop settings from an app.'
+  s.homepage         = 'https://github.com/spencerccf/app_settings'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = { 'ask.melnyk@gmail.com' => 'ask.melnyk@gmail.com' }
```
Comment (navaronbracke) [inline macos/app_settings.podspec:12]: This should refer to the author/publisher of the plugin, not your own email address.

See https://github.com/spencerccf/app_settings/blob/master/ios/app_settings.podspec#L12

```suggestion
  s.author           = { 'spencerccf@gmail.com' => 'spencerccf@gmail.com' }
```

**Review by navaronbracke — CHANGES_REQUESTED** [review]
One remark about a field in the podspec.

**Review by navaronbracke — APPROVED** [review]

**Comment by alex-melnyk** [comment]
@navaronbracke Anyway I can't merge it, please merge and publish the next version so I can use those changes from the https://pub.dev/ repository.

**Comment by navaronbracke** [comment]
@alex-melnyk you can use a git override as a workaround for now: https://dart.dev/tools/pub/dependencies#git-packages

cc @spencerccf since I do not have commit access.

**Comment by alex-melnyk** [comment]
> @alex-melnyk you can use a git override as a workaround for now: https://dart.dev/tools/pub/dependencies#git-packages
> 
> 
> 
> cc @spencerccf since I do not have commit access.

Yes, I already use that way, that is why I asking for the publishing a next version with my changes, thank you.

@spencerccf Please review changes, and publish a next version.

Best regards;)

**Comment by alex-melnyk** [comment]
> > @alex-melnyk you can use a git override as a workaround for now: https://dart.dev/tools/pub/dependencies#git-packages
> > cc @spencerccf since I do not have commit access.
> 
> Yes, I already use that way, that is why I asking for the publishing a next version with my changes, thank you.
> 
> @spencerccf Please review changes, and publish a next version.
> 
> Best regards 😉

@spencerccf any chance to look into this PR?

**Pull request merged**